ui/raidboss/data/06-ew/raid/p3s.ts

```diff
@@ -320,6 +334,89 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P3S Sunbird Tether',
+      type: 'Tether',
+      netRegex: NetRegexes.tether({ id: '0039' }),
+      promise: async (data) => {
+        const callData = await callOverlayHandler({
+          call: 'getCombatants',
+        });
+        if (!callData || !callData.combatants || !callData.combatants.length) {
+          console.error('SunbirdTether: failed to get combatants: ${JSON.stringify(callData)}');
+          return;
+        }
+        const sunbirds = callData.combatants.filter((c) => c.BNpcID === 13635);
+        if (sunbirds.length !== 4) {
+          console.error('SunbirdTether: There is not exactly four Sunbirds?!?: ${JSON.stringify(sunbirds)}');
+          return;
+        }
+        data.sunbirds = sunbirds;
+      },
+      alertText: (data, matches, output) => {
+        ++data.sunbirdTetherCounter;
+        if (data.sunbirdTetherCounter <= 8) {
+          // Save tether
+          if (matches.source === 'Sunbird') {
+            for (const s of data.sunbirds) {
+              if (s.ID && s.ID.toString(16).toUpperCase() === matches.sourceId) {
+                const pos = Math.round(2 - 2 * Math.atan2(s.PosX - 100, s.PosY - 100) / Math.PI) % 4;
```

Comment:
I know most folks do cards, but I think it'd be worth handling intercards here too just in case.

Reply:
This would reduce the leeway tanks have to put an add "north" significantly. Can the config GUI do checkboxes or something to let the user choose between two options for this?